addon/adapters/rest.js

```diff
@@ -67,7 +67,7 @@ const Promise = Ember.RSVP.Promise;
 
   ```js
   {
-    "post": {
+    "posts": {
```

Comment:
Will happily read "post" or "posts" when extracting the data but it will serialize requests as "posts" so I think its a good idea for the docs to show the pluralized version for consistency.